web/server/watcher.py

```diff
@@ -0,0 +1,32 @@
+import asyncio
+import json
+from pathlib import Path
+
+from watchfiles import awatch
+
+from web.server.api.endpoints.models import get_models
+from web.server.settings import get_loaded_context, get_settings
+from web.server.sse import Event
+
+
+async def watch_project(queue: asyncio.Queue) -> None:
+    settings = get_settings()
+    context = await get_loaded_context(settings)
+
+    def _get_path(path: str) -> str:
+        return str(Path(path).relative_to(context.path))
+
+    async for changes in awatch(settings.project_path):
```

Comment:
Should we only watch the directories SQLMesh cares about, e.g. audits, models, hooks, etc. We should also respect a project's IGNORE_PATTERNS so we don't trigger unnecessary reloads.

Reply:
@vchan  i changed it to only `watch` for models folder, because we going to change it anyway with multi repo support